kinesumer.go

```diff
@@ -54,19 +57,20 @@ var DefaultOptions = Options{
 	ShardAcquisitionTimeout: 90 * time.Second,
 }
 
-func NewDefault(stream string) (*Kinesumer, error) {
+func NewDefault(stream, duration string) (*Kinesumer, error) {
 	return New(
 		kinesis.New(session.New()),
 		nil,
 		nil,
 		nil,
 		stream,
 		nil,
+		duration,
 	)
 }
 
 func New(kinesis k.Kinesis, checkpointer k.Checkpointer, provisioner k.Provisioner,
-	randSource rand.Source, stream string, opt *Options) (*Kinesumer, error) {
+	randSource rand.Source, stream string, opt *Options, duration string) (*Kinesumer, error) {
```

Comment:
but i think the right option is to use `time.Duration`, i agree it's a better interface